iree/compiler/Codegen/SPIRV/KernelConfig.cpp

```diff
@@ -208,6 +208,10 @@ namespace detail {
 LogicalResult setMatmulOpConfig(linalg::LinalgOp op,
                                 std::array<int64_t, 2> bestWorkgroupSizeXY,
                                 std::array<int64_t, 3> bestThreadTileSizeMNK) {
+  auto lhsType = op.inputs()[0].getType().cast<ShapedType>();
+  auto elementBits = lhsType.getElementType().getIntOrFloatBitWidth();
+  if (elementBits != 16 && elementBits != 32) return success();
```

Comment:
So that means those types would be fully serialized? tiling should be fine, is the problem vectorization?

Reply:
Yes the problem is vectorization. They will still be distributed as it will fallback to another config in the following. The tests also showed that. The default distribute config is not considering multiple parallel dimensions though and that should be improved.